content/verification/va-letter-generator/release-notes/2025-06-09.md

```diff
@@ -0,0 +1,3 @@
+ - The VBA letterhead and logo has been restored to all letters except the Minimum Essential Coverage and Medicare Park D (which keep the VHA letterhead and logo). 
```

Comment:
I'm assuming Park is supposed to be Part? @lglover-va 

Also, could we get a quick intro to this release note? "We've made some changed to..." something like that